lib/internal/modules/cjs/loader.js

```diff
@@ -181,6 +182,13 @@ const {
   isProxy,
 } = require('internal/util/types');
 
+const { debuglog, debugWithTimer } = require('internal/util/debuglog');
+
+let { startTimer, endTimer } = debugWithTimer('module_cjs', (start, end) => {
```

Comment:
Should this be:

```suggestion
let { startTimer, endTimer } = debugWithTimer('module_timer', (start, end) => {
```

Instead? I feel that if we are to add something similar to ESM, there's not too much point to distinguish the two env vars. In the traces users can distinguish how the module get loaded with the log labels anyway.

Reply:
Hm.. makes sense to me, I initially thought the algorithm would accept if we pass `node.debuglog_module` to enable both `_cjs` and `_esm`, but I think it's not the case.